Index: app/src/main/java/com/example/datossesion/LoginActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/datossesion/LoginActivity.java	(revision f8bd763ea9ef31d00d453a5f689d8de18588dc80)
+++ app/src/main/java/com/example/datossesion/LoginActivity.java	(date 1654706881971)
@@ -9,13 +9,17 @@
 import android.widget.EditText;
 import android.widget.Toast;
 
+import java.sql.Connection;
+
 public class LoginActivity extends AppCompatActivity {
 
     private EditText txtUser, txtPwd;
-    private Button btnLogin;
+    private Button btnLogin, btnRegister;
     SessionManager session;
     DialogManager cuadroDialogo = new DialogManager();
 
+    Connection conn;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
@@ -26,6 +30,7 @@
         txtUser = findViewById(R.id.txtUserName);
         txtPwd =  findViewById(R.id.txtPassword);
         btnLogin = findViewById ( R.id.btnLogin );
+        btnRegister = findViewById ( R.id.btnRegister );
 
         if(session.isLogged()){
             txtUser.setVisibility ( View.INVISIBLE );
@@ -72,6 +77,14 @@
             Intent intent = new Intent(getApplicationContext(), MainActivity.class);
             startActivity(intent);
         }
+
+        btnRegister.setOnClickListener ( new View.OnClickListener ( ) {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(getApplicationContext(), RegisterActivity.class);
+                startActivity(intent);
+            }
+        } );
 
     }
 }
\ No newline at end of file
Index: app/src/main/java/com/example/datossesion/RegisterActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/datossesion/RegisterActivity.java	(date 1654706879515)
+++ app/src/main/java/com/example/datossesion/RegisterActivity.java	(date 1654706879515)
@@ -0,0 +1,14 @@
+package com.example.datossesion;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class RegisterActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate ( savedInstanceState );
+        setContentView ( R.layout.activity_register );
+    }
+}
\ No newline at end of file
Index: app/src/main/res/layout/activity_login.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_login.xml	(revision f8bd763ea9ef31d00d453a5f689d8de18588dc80)
+++ app/src/main/res/layout/activity_login.xml	(date 1654706656695)
@@ -28,4 +28,10 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:text="Ingresar" />
+
+    <Button
+        android:id="@+id/btnRegister"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Registrar" />
 </LinearLayout>
\ No newline at end of file
Index: app/src/main/java/com/example/datossesion/SessionManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/datossesion/SessionManager.java	(revision f8bd763ea9ef31d00d453a5f689d8de18588dc80)
+++ app/src/main/java/com/example/datossesion/SessionManager.java	(date 1654628324359)
@@ -58,7 +58,5 @@
         intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
         intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         _context.startActivity(intent);
-
-
     }
 }
Index: app/src/main/res/layout/activity_register.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_register.xml	(date 1654706879431)
+++ app/src/main/res/layout/activity_register.xml	(date 1654706879431)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".RegisterActivity">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/AndroidManifest.xml	(revision f8bd763ea9ef31d00d453a5f689d8de18588dc80)
+++ app/src/main/AndroidManifest.xml	(date 1654706879372)
@@ -9,7 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.DatosSesion">
-        <activity android:name=".CloseActivity"></activity>
+        <activity android:name=".RegisterActivity"></activity>
+        <activity android:name=".CloseActivity" />
         <activity
             android:name=".LoginActivity"
             android:exported="true">
@@ -19,7 +20,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".MainActivity"></activity>
+        <activity android:name=".MainActivity" />
     </application>
 
 </manifest>
\ No newline at end of file
